README.md

```diff
@@ -67,6 +67,23 @@ Plug 'nvim-treesitter/nvim-treesitter'
 Plug 'cuducos/yaml.nvim'
 ```
 
+## Winbar
+
+```lua
+
+-- set autocmd to get yaml path in winbar
+
+vim.api.nvim_create_autocmd({ "FileType" }, {
+	pattern = { "yaml" },
+	callback = function()
+  	vim.opt_local.winbar = "%{%v:lua.require'yaml_nvim'.get()%}"
+	end,
+})
+
+```
+
+![Example GIF WinBar](doc/demo.gif)
+
```

Comment:
I would remove the demo. It would be too much information in the landing page IMHO.

```suggestion
![Example GIF WinBar](doc/demo.gif)

```